drivers/mpl3115a2/include/mpl3115a2_reg.h

```diff
@@ -18,8 +18,8 @@
  *
  */
 
-#ifndef __MPL3115A2_REG_H__
-#define __MPL3115A2_REG_H__
```

Comment:
@Darredevil You have understood @Kijewski 's comment correctly. The other trailing double underscores or other double underscores were overlooked in the review process of your previous PRs (human error). If you want you can keep a single trailing underscore.